### DISCUSSION TITLE
IRS Tasking

### DISCUSSION THREAD

**Discussion by f-odhiambo** [opensrp/fhircore#485] [Ideas]
How do we intend to do IRS tasking based on the FHIR Spec? How does it work? How do we intend to improve it?

**Comment by pld** [comment]
$apply spec can take a `Location` resource as a subject, BUT I don't think this is supported by the Android library yet, also not sure if it's supported by the server

Let's understand timeline for Location as subject working w/$apply and understand if we need an alternative
